<testsuites id="" name="" tests="1" failures="1" skipped="0" errors="0" time="15.33281700000004">
<testsuite name="e2e\employees\add-employee-success.spec.ts" timestamp="2025-03-14T20:01:34.374Z" hostname="chromium" tests="1" failures="1" skipped="0" time="10.985" errors="0">
<testcase name="[US02] Gestión de empleados | Agregar nuevos empleados › Escenario 1 | Validar agregar un empleado exitosamente" classname="e2e\employees\add-employee-success.spec.ts" time="10.985">
<failure message="add-employee-success.spec.ts:10:9 Escenario 1 | Validar agregar un empleado exitosamente" type="FAILURE">
<![CDATA[  [chromium] › e2e\employees\add-employee-success.spec.ts:10:9 › [US02] Gestión de empleados | Agregar nuevos empleados › Escenario 1 | Validar agregar un empleado exitosamente › Entonces el empleado se encuentra en la lista de empleados 

    Error: Locator for ID 875 not found.

      41 |                 await expect(locator).toBeVisible();
      42 |             } else {
    > 43 |                 throw new Error(`Locator for ID ${uniqueID} not found.`);
         |                       ^
      44 |             }
      45 |         });
      46 |     });
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\employees\add-employee-success.spec.ts:43:23
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\employees\add-employee-success.spec.ts:32:9
]]>
</failure>
</testcase>
</testsuite>
</testsuites>